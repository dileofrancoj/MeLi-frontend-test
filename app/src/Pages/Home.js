import React, { Fragment } from "react";
import Helmet from "react-helmet";
import Message from "./../Components/Message";
const Home = () => {
  return (
    <Fragment>
      <Helmet>
        <title>Mercado Libre Argentina</title>
        <meta name="description" content="EncontrÃ¡ lo que buscas" />
      </Helmet>
      <Message message={"Buscar productos ..."} />
    </Fragment>
  );
};

export default Home;
