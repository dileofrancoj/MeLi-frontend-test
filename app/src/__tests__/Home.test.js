import React from "react";
import { shallow } from "enzyme";
import { StaticRouter } from "react-router-dom";
import Home from "../Pages/Home";

describe("Home", () => {
  test("should render Home component ", () => {
    const component = shallow(
      <StaticRouter>
        <Home />
      </StaticRouter>
    );
    expect(component).toMatchSnapshot();
  });
});
