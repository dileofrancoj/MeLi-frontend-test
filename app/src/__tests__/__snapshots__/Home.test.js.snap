// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render Home component  1`] = `
<Router
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
  staticContext={Object {}}
>
  <Home />
</Router>
`;
